src/machines/builder_route/axis.config.js

```diff
@@ -0,0 +1,140 @@
+export const axis_config = {
+	id: "Axis",
+	initial: "CheckingAxis",
+	on: {
+		'SELECTION_CHANGED': {
+			target: "Axis.CheckingAxis",
+			actions: ['setURL']
+		},
+		"AXIS_RESET": {
+			target: "Axis.AxisIncomplete",
+			actions: [
+				'deleteChildAxes',
```

Comment:
I was aiming to express the relation between `addChildAxis` and this action by naming it similarly. I'll rename this as well to `addNestedAgg`

Reply:
Perhaps we should also rename the `Axis` machine to `Aggregation` machine.

Reply:
That was my first thought by it would be confuging because "aggregation" is already used for the name of the aggregation itself and it can be intended as the aggregation query.
I'd prefer something else than Axis, but nothing came to mind.

Rather than an aggregation machine (an aggregation doesn't have states) it would be more of a `Selection` machine: a selection can be complete or incomplete, and it can be in a `checking` state, so it would make sense to me.

Reply:
The selection however refers exclusively to an aggregation configuration, so perhaps `AggSelection` ?

Reply:
See https://github.com/nestauk/dapsboard/pull/70#issuecomment-672861320